Userland/Libraries/LibJS/Bytecode/ASTCodegen.cpp

```diff
@@ -129,7 +129,7 @@ Bytecode::CodeGenerationErrorOr<void> ScopeNode::generate_bytecode(Bytecode::Gen
             // Note: Already done in step iv. above.
 
             // 4. Insert d as the first element of functionsToInitialize.
-            functions_to_initialize.prepend(function);
+            functions_to_initialize.append(function);
```

Comment:
Needs a `// NOTE: ...` below the comment explaining why we ignore the spec's instruction.